src/main/resources/templates/common.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+
+<head th:fragment="styling">
+    <link th:href="@{/css/reset.css}" rel="stylesheet" type="text/css">
+    <link th:href="@{/css/style.css}" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<nav th:fragment="navbar">
+    <a th:href="@{/}">Trivia</a>
+    <a th:if="${user != null}" th:href="@{/admin/schedule}">Schedule</a>
```

Comment:
Another good use of the th:if!

You could have 'grouped' these into a span, though, so you didn't have to repeat it on each entry.

(That said, if you updated the app in the future to use different roles for different features, you might just have added them back)